src/app/shell/main/filters-list/city-filter/city-filter.component.html

```diff
@@ -0,0 +1,15 @@
+<form>
+    <mat-icon> place</mat-icon>
+    <mat-form-field >
+        <input type="text" class="filter__metaData-city"
```

Comment:
needs to specify which css naming convention to use. Let's discuss if it worth to use BEM